fancylog/fancylog.py

```diff
@@ -37,6 +37,7 @@ def start_logging(
     log_to_file=True,
     log_to_console=True,
     timestamp=True,
+    logger_name=False,
```

Comment:
False as a default seems weird to me. I would normally have a default as `None`. Is there a reason to prefer one or another?

Reply:
Agreed, I think at the time I was imagining 'Am I passing a logger_name? FALSE' but 'None' makes much more sense (also `True` is not a valid input).